images/converter/default.go

```diff
@@ -206,7 +264,7 @@ func (c *defaultConverter) convertManifest(ctx context.Context, cs content.Store
 		return nil, err
 	}
 
-	newConfig, err := c.convert(ctx, cs, manifest.Config)
+	newConfig, err := c.convertConfig(ctx, cs, manifest.Config, dm)
```

Comment:
It seems that the `docker2oci` is ignored for config blob.
Conversion should always invoked via `convert` function to ensure that the hook is called and media type conversion happens.

Reply:
Yes, also aware of this problem, but do we need to fix it in this PR? If so I can add a new commit for this.

Reply:
Fixed, thanks for reviewing.